libraries/AP_DAL/AP_DAL.cpp

```diff
@@ -99,6 +101,15 @@ void AP_DAL::start_frame(AP_DAL::FrameType frametype)
 #endif
 }
 
+// for EKF usage to enable takeoff expected
```

Comment:
... or maybe we should change this comment to "set takeoff expected to true"

Reply:
done